include/cbl++/Database.hh

```diff
@@ -100,9 +100,15 @@ namespace cbl {
             check(CBLDatabase_Delete(ref(), &error), error);
         }
 
+        /** Deprecated, use performMaintenance with kCBLMaintenanceTypeCompact instead. */
```

Comment:
No need to deprecate in an unsupported product.  Removing should be fine.  If you really want to deprecate then use deprecation attributes (LiteCore should have a reference)

Reply:
Done. I removed compact() from C++ API.

Reply:
I still see deprecations though?

Reply:
Are you looking at C or C++? It has been removed from c++ header, but deprecated in C.

Reply:
@borrrden let's not remove compact() from C-API for now as they are used by the other bindings in the repo that I don't want to make any changes to that yet.

Reply:
OK, I removed it.